api/autopilot.go

```diff
@@ -39,7 +39,6 @@ type (
 	AutopilotConfig struct {
```

Comment:
Blocker to avoid merging this for now. It requires a `hostd` change to be released first. Namely v0.3.0